catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/OrderRequestImpl.java

```diff
@@ -96,23 +85,45 @@
 
     private static final int MAX_MEMORY_SIZE = 100 * MB;
 
+    private final Function<Destination, Optional<DestinationSink>> destinationSinkFactory;
+
     private OrderContents order;
 
     private AccessManagerImpl accessManager;
 
     private CatalogFramework catalogFramework;
 
-    private String protocol;
-
-    private int port;
-
-    public OrderRequestImpl(OrderContents order, String protocol, int port,
-            AccessManagerImpl accessManager, CatalogFramework catalogFramework) {
+    /**
+     * This constructor is only intended for unit testing.
```

Comment:
unfortunately, they class and the unit tests are in different packages. I might be able to fix that, though.